config/routes.rb

```diff
@@ -79,10 +79,8 @@
       end
 
       namespace 'rh_cloud' do
-        unless ForemanRhCloud.with_local_advisor_engine?
-          post 'enable_connector', to: 'inventory#enable_cloud_connector'
-          post 'cloud_request', to: 'cloud_request#update'
-        end
+        post 'enable_connector', to: 'inventory#enable_cloud_connector'
```

Comment:
do we need to allow enabling cloud connector for iop ?

Reply:
You can ignore. Looks like https://github.com/theforeman/foreman_rh_cloud/pull/1049 is already handling this